src/Compilers/CSharp/Portable/Binder/Semantics/Conversions/ConversionKind.cs

```diff
@@ -56,5 +56,7 @@ internal enum ConversionKind : byte
         PinnedObjectToPointer,
 
         DefaultLiteral, // a conversion from a `default` literal to any type
+
+        ImplicitPointer,
```

Comment:
>ImplicitPointer [](start = 8, length = 15)

How does it relate to ```PointerToPointer```? Consider adding comments to both to make sure the distinction is clear.

Reply:
Will do. `PointerToPointer` is for explicit pointer conversions.

Reply:
> PointerToPointer is for explicit pointer conversions.

Consider renaming to reflect that fact.

---
In reply to: [402645414](https://github.com/dotnet/roslyn/pull/43007#discussion_r402645414) [](ancestors = 402645414)

Reply:
Perhaps also placing them next to each other.

---
In reply to: [402646040](https://github.com/dotnet/roslyn/pull/43007#discussion_r402646040) [](ancestors = 402646040,402645414)

Reply:
Well, PointerToPointer is the name given in the spec:

> Additionally, in an unsafe context, the set of available explicit conversions (Explicit conversions) is extended to include the following explicit pointer conversions:
>    * From any pointer_type to any other pointer_type.

Reply:
I can add the word `Explicit` to it though.